service/src/test/java/uk/gov/hmcts/reform/fpl/service/docmosis/EPOGenerationServiceTest.java

```diff
@@ -92,6 +159,44 @@ CaseData getCase(OrderStatus orderStatus) {
             .build();
     }
 
+    CaseData getEpoExclusionRequirementCase(OrderStatus orderStatus,
+                                            EPOExclusionRequirementType epoExclusionRequirementType,
+                                            LocalDate epoExclusionStartDate,
+                                            String whoIsExcluded) {
+        return defaultCaseData(orderStatus)
+            .dateOfIssue(null)
+            .dateAndTimeOfIssue(time.now())
+            .orderTypeAndDocument(OrderTypeAndDocument.builder()
+                .type(EMERGENCY_PROTECTION_ORDER)
+                .document(DocumentReference.builder().build())
+                .build())
+            .epoChildren(EPOChildren.builder()
+                .descriptionNeeded("Yes")
+                .description("Test description")
+                .build())
+            .epoEndDate(time.now().plusDays(5))
+            .epoPhrase(EPOPhrase.builder()
+                .includePhrase("Yes")
+                .build())
+            .epoType(REMOVE_TO_ACCOMMODATION)
```

Comment:
maybe change epo type to prevent removal because exclusion is only shown when this radio is selected

Reply:
sorted and added additional check related to REMOVE_TO_ACCOMODATION